app/policies/group_policy.rb

```diff
@@ -254,19 +254,19 @@ def destroy_attachment?
   end
 
   def create_metadata_template?
-    return true if effective_access_level >= Member::AccessLevel::ANALYST
+    return true if effective_access_level >= Member::AccessLevel::MAINTAINER
```

Comment:
This and the other policy changes can be updated to `return true if Member::AccessLevel.manageable.include?(effective_access_level)`

Reply:
Updated in 8c70e48a457013e1de19b9b78ce52bee917ce5ea

Reply:
Updated in 8c70e48a457013e1de19b9b78ce52bee917ce5ea